pkg/crc/machine/bundle/repository.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	supportedVersion = "^1.0"
-	bundleExtension  = ".crcbundle"
+	Extension        = ".crcbundle"
```

Comment:
If `GetCustomBundleName` returns a bundle name with its extension, and with the help of `GetBundleNameWithoutExtension`, I think you can achieve the same as what you do in this commit without exporting `Extension`.

Reply:
@cfergeau it return name without extension, I fixed the naming of the function as mentioned https://github.com/code-ready/crc/pull/2147#discussion_r620999485 here.

Reply:
If you don't change the naming, but return a name with the extension instead, you can reuse existing API. Here you are adding additional API which overlaps with API added in this PR.

Reply:
@cfergeau updated.